io/net/virt-net/network_virtualization.py

```diff
@@ -172,6 +156,20 @@ def run_command(self, command, timeout=300):
         con.prompt(timeout)
         return output
 
+    def check_slot_availability(self):
+        '''
+        Checks if given slot is available.
+        Returns True if yes.
```

Comment:
can doc string modified  like as 
`:return: True if slot available other wise false `

can this utility target to avocado lib

Reply:
sure